libnmstate/nm/user.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+""" Use a Network Manager User Setting to store and retrieve a device
+description:
+
+https://lazka.github.io/pgi-docs/#NM-1.0/classes/SettingUser.html
+"""
+
+from libnmstate.nm import nmclient
+
+NMSTATE_DESCRIPTION = "nmstate.description"
```

Comment:
This is interface specific, we may need other descriptions in the future.

Reply:
changed to `nmstate.interface.desciption`